src/app/analytics.service.spec.ts

```diff
@@ -47,6 +47,17 @@ describe('AnalyticsService', () => {
         regulatorNumber: '123456',
         regulatorRegion: 'Scotland',
       },
+      {
+        charityOptIn: 'Charity opt in',
```

Comment:
@tbg-jasonfung what's going on with all of these?!

Reply:
@NoelLH it's custom labels I mentioned [here](https://thebiggive.slack.com/archives/DCJLDGN2J/p1603095146002400)

Reply:
@tbg-jasonfung these are still not documented per-value anywhere I can see, and it doesn't seem like the examples are realistic so the tests don't help at all in understanding the purposes.

Reply:
@NoelLH this might help paint a better picture: https://github.com/thebiggive/donate-frontend/pull/332/files